.github/labeler.yml

```diff
@@ -0,0 +1,34 @@
+dependencies:
+- changed-files:
+  - any-glob-to-any-file:
+    - "requirements.txt"
+    - "setup.py"
+    - "pyproject.toml"
+    - "Pipfile"
+    - "Pipfile.lock"
+    - "requirements/*.txt"
+    - "requirements/*.in"
+
+documentation:
+- changed-files:
+  - any-glob-to-any-file:
+    - "docs/**/*"
+    - "*.md"
+    - "*.rst"
+
+config:
+- changed-files:
+  - any-glob-to-any-file:
+    - ".pre-commit-config.yaml"
+    - "config/**/*"
+    - "settings/**/*"
+    - "*.ini"
+    - "*.cfg"
+    - "*.conf"
+
+CI/CD:
+- changed-files:
+  - any-glob-to-any-file:
+    - ".github/**/*"
+    - "tox.ini"
+    - ".coveragerc"
```

Comment:
should we add "test/**/*" here?

Reply:
Happy to add a tests label instead. What do you think?